addons/website_crm_partner_assign/static/src/js/crm_partner_assign.js

```diff
@@ -277,9 +282,9 @@ publicWidget.registry.crmPartnerAssign = publicWidget.Widget.extend({
 
     _parse_date: function (value) {
         console.log(value);
-        var date = moment(value, time.getLangDateFormat(), true);
-        if (date.isValid() && date.year() >= 1900) {
-            return time.date_to_str(date.toDate());
+        var date = parseDate(value);
+        if (date.isValid && date.year >= 1900) {
+            return formatDate(date, { format: localization.timeFormat });
```

Comment:
nope, here you should format to a date, not time
just do not pass any option to formatDate